feature-libs/product/image-zoom/components/product-image-zoom/product-image-zoom-product-images/product-image-zoom-product-images.component.html

```diff
@@ -9,6 +9,11 @@
 
 <ng-container *ngIf="thumbs$ | async as thumbs">
   <cx-carousel
+    role="region"
+    [attr.aria-label]="
+      'carousel.carouselForProduct' | cxTranslate: { product: getProductName() }
+    "
+    tabindex="-1"
```

Comment:
The `cx-carousel` itself shouldn't be accessible by tab key. Is there an attribute that makes it accessible?

Reply:
Well the `tabindex="-1"` means that the element should be focusable, but should not be reachable via sequential keyboard navigation. As for `role="region`" it does not modify keyboard navigation either.

Does this answer the question?